apps/myaccount/src/layouts/dashboard.tsx

```diff
@@ -0,0 +1,303 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Can we update the license header

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3440/commits/d16981abb35182947ec076270163483cb6bd55f6